dojo/unittests/tools/test_gitlab_dep_scan_parser.py

```diff
@@ -21,6 +21,14 @@ def test_parse_file_with_one_vuln_has_one_finding(self):
         findings = parser.get_findings(testfile, Test())
         self.assertEqual(1, len(findings))
 
+    def test_parse_file_with_one_vuln_missing_component_has_one_finding(self):
+        testfile = open(
+            "dojo/unittests/scans/gitlab_dep_scan/gl-dependency-scanning-report-1-vuln-missing-component.json"
+        )
+        parser = GitlabDepScanReportParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(1, len(findings))
+
```

Comment:
Could you have something like that?
```suggestion
        finding = findings[0]
        self.assertEqual("...", finding.component_name)
        self.assertEqual("...", finding.component_version)
        <... mode tests? ...>
```

Reply:
Can you please review @damiencarol ?
Hope that I understood what you asked for. Otherwise, let me know if it needs improvement (I'm not very proficient in Python 😬).

Reply:
it's good but you only check 1 finding without component data.

Reply:
add another one with component data plz

Reply:
Done! One finding has no components and the other has both name and version. PTAL.